components/Button/Button.style.js

```diff
@@ -1,48 +1,54 @@
 import { styled } from "styled-components";
 
-export const ButtonStyled = styled.button`
+export const ButtonBody = styled.button`
   ${(props) =>
-    props.name === "back" &&
-    `
-  background-color: var(--mainDarkColor);
-  color: var(--mainLightColor);
-  font-size: 1.5em;
-  border: none;
-  position: absolute;
-  left: 10px;
-  `};
+    props.variant === "big"
+      ? "border: solid;  border-radius: 25px;  background-color: var(--mainLightColor);  color: var(--mainDarkColor);  position: relative;  height: 3em;  margin: 20px auto;  display: flex;  align-items: center;  justify-content: flex-end;"
+      : ""}
```

Comment:
is there a reason you write it that way? If you `import { styled, css } from "styled-components"` you can write it like that:
props.variant === "big" && css`` (between the backticks write normal css)

This way your css is much easier to read.